com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Window.cs

```diff
@@ -137,18 +138,30 @@ public ConfigStyle(string label, string error, string button = resolve, MessageT
             public static readonly ConfigStyle dxrTransparentReflections = new ConfigStyle(
-                label: "Transparent Reflections",
-                error: "Transparent Screen Space Reflections are disabled in the current HDRP asset. You will not be able to toggle ray traced reflections on transparent objects though your volume components. You can enable the feature in the HDRP asset under Lighting -> Reflections -> Transparent Screen Space Reflections", messageType: MessageType.Info);
+                label: "Transparent Reflection (Asset)",
+                error: "Transparent Screen Space Reflection is disabled in the current HDRP asset. You will not be able to toggle ray traced reflections on transparent objects though your volume components. You can enable the feature in the HDRP asset under Lighting > Reflections > Transparent Screen Space Reflections", messageType: MessageType.Warning);
+            public static readonly ConfigStyle dxrTransparentReflectionsFS = new ConfigStyle(
+                label: "Transparent Reflection (Default Camera Frame Setting)",
+                error: "Transparent Screen Space Reflection is disabled in the default Camera Frame setting. You can enable the feature in Project Settings > HDRP Default Settings > Frame Settings > Default Frame Settings For Camera > Lighting > On Transparent", messageType: MessageType.Info);
             public static readonly ConfigStyle dxrGI = new ConfigStyle(
-                label: "Global Illumination",
-                error: "Screen Space Global Illumination is disabled in the current HDRP asset. You will not be able to toggle ray global illumination though your volume components. You can enable the feature in the HDRP asset under Lighting -> Screen Space Global Illumination", messageType: MessageType.Info);
-			public static readonly ConfigStyle dxr64bits = new ConfigStyle(
+                label: "Global Illumination (Asset)",
+                error: "Screen Space Global Illumination is disabled in the current HDRP asset. You will not be able to toggle ray global illumination though your volume components. You can enable the feature in the HDRP asset under Lighting -> Screen Space Global Illumination", messageType: MessageType.Warning);
+            public static readonly ConfigStyle dxrGIFS = new ConfigStyle(
+                label: "Global Illumination (Default Camera Frame Setting)",
+                error: "Screen Space Global Illumination is disabledin the default Camera Frame setting. You can enable the feature in Project Settings > HDRP Default Settings > Frame Settings > Default Frame Settings For Camera > Lighting > Screen Space Global Illumination", messageType: MessageType.Info);
```

Comment:
Screen Space Global Illumination is disabled in the default Camera Frame Settings. This means Cameras that use these Frame Settings do not render ray-traced global illumination. To enable this feature, go to **Project Settings > HDRP Default Settings > Frame Settings > Default Frame Settings For Camera > Lighting** and enable **Screen Space Global Illumination**.